src/StatusQ/Layout/StatusMainLayout.qml

```diff
@@ -0,0 +1,34 @@
+import QtQuick 2.13
+import QtQuick.Layouts 1.13
+import QtQuick.Controls 2.13
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+
+SplitView {
+    id: statusAppLayout
+
+    implicitWidth: 900
+    implicitHeight: 600
+
+    property Item leftPanel
+    property Item rightPanel
+
+    handle: Item { }
```

Comment:
`null`

Reply:
null crashes, undefined breaks in 
```
    color: T.SplitHandle.pressed ? control.palette.mid : (T.SplitHandle.hovered ? control.palette.midlight : control.palette.button)
```
in `/qml/QtQuick/Controls.2/SplitView.qml`, so we better leave it as is

Reply:
Ok, looks like a bug in QQC2.Templates :( ignore then